src/client/components/serach-summary/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './index.scss';
+import {Header} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+
+export class SearchSummaryComponent extends React.Component {
+    static defaultProps = {
+        numberOfMatched: 0
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+
+            // destination: this.props.destination,
+            // numberOfMatched: this.props.numberOfMatched
+
+
+            //stories mock
+            destination: this.props.data.destination,
```

Comment:
There is no need to duplicate properties in state. Your component doesn't need local state, only props.